### DISCUSSION TITLE
Store Analytics endpoint?

### DISCUSSION THREAD

**Discussion by tilschuenemann** [denisneuf/python-amazon-ad-api#190] [Q&A]
I've searched the docs, but wasn't sure if this library provides means to access the Store Analytics, eg. the getAsinEngagementForStore endpoints.

Are these scoped in?

**Comment by denisneuf** [comment]
Hi @tilschuenemann can you point the url of the Amazon Ads API official endpoint that you want to be added?

**Reply by tilschuenemann** [reply]
The endpoint should be `/stores/{brandEntityId}/asinMetrics` per [docs](https://advertising.amazon.com/API/docs/en-us/stores/open-api#tag/Stores-Analytics/operation/getAsinEngagementForStore).

**Comment by denisneuf** [comment]
Hi @tilschuenemann can you point the url of the Amazon Ads API official endpoint that you want to be added?
